www/viewgame

```diff
@@ -1432,20 +1437,20 @@ if (!isEmpty($seriesname)) {
         echo "Part of ";
     else
         echo "Episode $seriesnum of ";
-    echo "<a href=\"search?searchfor=series:" . urlencodeFromHTML($seriesname)
-        . "\">$seriesname</a><br>";
+    echo "<a href=\"search?searchfor=series:" . urlencode($seriesname)
```

Comment:
I don't remember how the search "unpacks" this data in reverse. It's not enough to `urlencode` this? It has to also be HTML-escaped?

Reply:
I'm not sure if you answered exactly what I was asking.

Now it's basically `urlencode(htmlspecialchars($raw_seriesname))` - is that the intention? Just to pass the data through the URL medium, it's enough to use `urlencode()` with `%XX`. Does the other side (search) expect it to also be HTML-escaped with `&XXX;`?

Reply:
I claim that line 1440 is now calling `urlencode($raw_seriesname)`, and then in the body of the `<a>` tag we're displaying `htmlspecialchars($raw_seriesname)`.

The other side (search) no longer expects or requires the data to be HTML-escaped. Search just pulls out the `$seriesname` raw from the `$_REQUEST` and uses it.

Reply:
Oh, sorry, I missed that `$seriesname` was not in the list above. OK :)